src/app/components/List/AuthorsList.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Html5Entities } from 'html-entities';
+import { Link } from 'react-router';
+
+const htmlEntities = new Html5Entities();
+
+const getIdentifier = author => (author.viaf && 'viaf') || (author.lcnaf && 'lcnaf') || 'name';
+
+const linkToAuthor = author => ({
+  q: author[getIdentifier(author)],
+  field: getIdentifier(author),
+  showQuery: `"${author.name}"`,
+  showField: 'author',
+});
+const birthDate = entity => (
+  <span>
+    {(entity.birth_date_display || entity.death_date_display) && <span> (</span>}
+    {entity.birth_date_display && <span>{entity.birth_date_display}</span>}
+    {entity.death_date_display && (
+      <span>
+        {' '}
```

Comment:
Hmm I'm not liking these.

Reply:
Fixed all of this (the formatter kept doing that)